src/odo.ts

```diff
@@ -389,7 +389,7 @@ export interface Odo {
     getClusters(): Promise<OpenShiftObject[]>;
     getProjects(): Promise<OpenShiftObject[]>;
     loadWorkspaceComponents(event: WorkspaceFoldersChangeEvent): void;
-    addWorkspaceComponent(WorkspaceFolder: WorkspaceFolder, component: OpenShiftObject);
+    addWorkspaceComponent(WorkspaceFolder: WorkspaceFolder, component: OpenShiftObject): void;
```

Comment:
If you want to improve, clean up, fix issues like this, you are welcome to send additional PR. This PR should have changes to compiler or linter settings to report issues like this during the build.
Please remove it form this PR.